src/plot/svgMask.tsx

```diff
@@ -0,0 +1,139 @@
+import { SvgRect } from '@h5web/lib';
+import type { SVGProps } from 'react';
+import { Vector3 } from 'three';
+
+type Rect = [Vector3, Vector3];
+
+interface Props extends SVGProps<SVGPathElement> {
+    coords: Rect;
+    strokePosition?: 'inside' | 'outside';
```

Comment:
could: consider making it a boolean? or enum

Reply:
This was to match SvgRect in h5web/lib.

https://github.com/silx-kit/h5web/blob/main/packages/lib/src/interactions/svg/SvgRect.tsx